pandas/core/groupby.py

```diff
@@ -191,6 +191,107 @@
     dtype: int64
     """)
 
+_numeric_operations_full_names = dict(
+    sum='the sum',
+    prod='the product',
+    min='the minimum',
+    max='the maximum',
+    first='the first value',
+    last='the last value'
+)
+
+_numeric_operations_doc_template = dedent(
+    """
+    Compute %(full_name)s of each group.
+
+    For multiple groupers, the result index will be a
+    :class:`~pandas.MultiIndex`.
+
+    Parameters
+    ----------
+    kwargs : dict
+        Optional keyword arguments to pass to `%(f)s`.
+
+        `numeric_only` : bool
+            Include only float, int, boolean columns.
```

Comment:
I think it would be fine to actually list them as normal parameters (not inside the description of the 'kwargs').

I am only not sure if `min_count` is used for all. I think it is only used for sum and prod. 
So not sure what the best way is here. Ideally this would also be substituted into the template, but that might get complicated.

Reply:
(given that this can get complicated, it is also fine to leave this for a separate issue/PR, and not solve it here directly)

Reply:
Cool @jorisvandenbossche :+1: I'll create a new issue for the complicated template substitution. I had a question about the Parameters - listing the parameters like this:
```
    Parameters
    ----------
    kwargs : dict
        For compatibility with other groupby methods.
    numeric_only : bool
        Include only float, int, boolean columns.
    min_count : int
        The required number of valid values to perform the operation.
        If fewer than `min_count` non-NA values are present the result
        will be NA.
```
results in errors when I run the `scripts/validate_docstrings.py` script:
```
Errors found:
        Errors in parameters section
                Unknown parameters {'numeric_only', 'min_count'}
```

Is this alright?